docs/Technical-Documentation/Architecture-Decision-Record/018-versioning-and-releases.md

```diff
@@ -0,0 +1,59 @@
+# 18. Versioning and Release strategy
+
+Date: 2022-04-05 (yyyy-mm-dd)
+
+## Status
+
+Accepted
+
+## Context
+As discussed briefly in [deployment-flow](./008-deployment-flow.md), we will need a strategy for our versioning and releases.
+
+## Decision
+
+### Gitflow
+We will use the industry standards "gitflow" to handle release branching above our [git-workflow](./009-git-workflow.md). You can read more about gitflow [here](https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow) and [here](https://datasift.github.io/gitflow/IntroducingGitFlow.html). As shown below, the raft team will merge any relevant feature branches of the form `feat/###-branch-description` into the `develop` branch which will deploy to staging sites. From this branch, we will split off a release-specific branch of the form `release/v#.#.#`. Once this release is ready, it will be merged to `HHS/TANF-app:master` which should contain only finalized merge commits of the release versions and will deploy to production.
```

Comment:
@abottoms-coder `HHS/TANF-app:master` already exists, so may need to be caught up before we can merge to it (https://github.com/HHS/TANF-app/tree/master) 

also can you remind happens to `hhs:main` going forward?

Reply:
Good questions! I should've addressed this specifically. As we have been having hhs:main mirror raft-tdp-main, I assumed that would continue. As for master, we should simply update with the `release/v1.0.0` merge.

Reply:
@abottoms-coder, I'd like to add this ADR to the agenda for tomorrow's (4/8) tech sync. Still thinking through whether staging should reflect all the latest features or only the release features. Ideally, we'd want to stage the release-specific production context. Is that a possibility here? cc: @lfrohlich @stevenino

Reply:
Let's discuss at tech sync. 👍